addons/website/controllers/main.py

```diff
@@ -535,7 +536,13 @@ def make_scss_custo(self, url, values):
         Returns:
             boolean
         """
-        request.env['web_editor.assets'].make_scss_customization(url, values)
+        # Protect variable names so they cannot be computed as numbers on SCSS
+        # compilation (e.g. var(--700) => var(700)).
+        scss_fixed_values = {key: re.sub(r"var\(--([0-9]+)\)",
+            lambda matchobj: "var(--#{" + matchobj.group(1) + "})",
+            value) for key, value in values.items()}
```

Comment:
This line triggered my linter and I think it's worth applying what is suggested as it's a bit tricky without it
<img width="529" alt="CleanShot 2022-12-16 at 15 30 15@2x" src="https://user-images.githubusercontent.com/77747125/208120530-1df09a71-03cc-4a80-b667-3b2a80ac9877.png">

Here's what I would at least do:

```suggestion
        scss_fixed_values = {
            key: re.sub(r"var\(--([0-9]+)\)",
                        lambda matchobj: "var(--#{" + matchobj.group(1) + "})", value)
            for key, value in values.items()
        }
```
So that it's clear where the regex starts and end, and where the population of the dict is. Though personally I think I would have preferred a classic for loop in a case like this for clarity. If you can, at least, apply the line indentation suggestion, it would be great 😄

Reply:
`for` loop is better indeed :+1: